cryptographic_estimators/SDEstimator/SDAlgorithms/bjmm.py

```diff
@@ -64,7 +68,7 @@ def __init__(self, problem: SDProblem, **kwargs):
 
     def initialize_parameter_ranges(self):
         """
-        initialize the parameters p, l, p1 and for d=3 p2
+        Initialize the parameters p, l, p1, and for d=3, p2.
```

Comment:
this docstring was wrong before already, it should say "Initialize parameter range for d"